planning/planning_scene_monitor/include/moveit/planning_scene_monitor/planning_scene_monitor.h

```diff
@@ -278,7 +278,7 @@ class PlanningSceneMonitor : private boost::noncopyable
 
   /** @brief Update the scene using the monitored state. This function is automatically called when an update to the current state is received (if startStateMonitor() has been called).
       The updates are throttled to a maximum update frequency however, which is set by setStateUpdateFrequency(). */
-  void updateSceneWithCurrentState();
+  void updateSceneWithCurrentState( bool triggerSceneUpdate= true);
```

Comment:
:-1: on changing the signature of updateSceneWithCurrentState.
As far as I can see, there is no need for this change to address the issue at hand,
it makes the actual patch much more confusing, and it would break the ABI.